src/main/java/uk/co/ramp/Outbreak.java

```diff
@@ -28,15 +25,18 @@
 public class Outbreak {
 
     private static final Logger LOGGER = LogManager.getLogger(Outbreak.class);
-    public static final String INFECTION_MAP = "infectionMap.txt";
 
     private StandardProperties properties;
     private DiseaseProperties diseaseProperties;
     private RandomDataGenerator rng;
-    private int tab = 0;
+
     private Map<Integer, Case> population;
     private Map<Integer, List<ContactRecord>> contactRecords;
+    private final LogDailyOutput outputLog = new LogDailyOutput();
```

Comment:
For consistency, would it be better to make LogDailyOutput() a bean and inject this in instead?

Reply:
Done